crates/gpui/src/window.rs

```diff
@@ -3432,25 +3432,25 @@ impl Window {
     /// This method should only be called as part of the paint phase of element drawing.
     pub fn on_mouse_event<Event: MouseEvent>(
         &mut self,
-        mut handler: impl FnMut(&Event, DispatchPhase, &mut Window, &mut App) + 'static,
+        mut listener: impl FnMut(&Event, DispatchPhase, &mut Window, &mut App) + 'static,
     ) {
         self.invalidator.debug_assert_paint();
 
         self.next_frame.mouse_listeners.push(Some(Box::new(
             move |event: &dyn Any, phase: DispatchPhase, window: &mut Window, cx: &mut App| {
                 if let Some(event) = event.downcast_ref() {
-                    handler(event, phase, window, cx)
+                    listener(event, phase, window, cx)
                 }
             },
         )));
     }
 
-    /// Register a key event listener on the window for the next frame. The type of event
+    /// Register a capturing key event listener on this node for the next frame. The type of event
```

Comment:
This seems incorrect? `on_key_event` and `on_mouse_event` are non capturing by default:

https://github.com/zed-industries/zed/blob/7b6ef5eaabc3755515170709b3e880075b86a69b/crates/gpui/src/window.rs#L3912-L3919

Reply:
I guess this is true; the listener will be triggered during both the capture and bubble phases. Updating the wording.